### PR TITLE
Convert i & l files ts

### PR DESCRIPTION
**Pull request by vdusart**
Moving files starting with letter "I" and "L" to TS

# Related Issue

Conversion to TS according to #6392

### CODE REVIEW

File: src/components/ImageCard.tsx [hunk 1]
```diff
@@ -21,7 +21,20 @@ const Description = styled.p`
   opacity: 0.8;
 `
 
-const ImageCard = ({ image, title, description, children, className }) => (
+export interface IProps {
+  image: string
+  title: string
+  description: string
+  className?: string
```
Comment (minimalsm) [inline src/components/ImageCard.tsx:28]: We should pass alts to images explicitly :-)

Comment (vdusart) [reply src/components/ImageCard.tsx:29]: Yeah good idea, just added it

File: src/components/Layout.tsx [hunk 2]
```diff
@@ -60,10 +60,39 @@ const Main = styled.main`
   flex-grow: 1;
 `
 
-const Layout = (props) => {
-  const [isDarkTheme, setIsDarkTheme] = useState(false)
-  const [isZenMode, setIsZenMode] = useState(false)
-  const [shouldShowSideNav, setShouldShowSideNav] = useState(false)
+interface PageContext {
```
Comment (pettinarip) [inline src/components/Layout.tsx:63]: There is already a `Context` defined in `src/types`. Lets use that one here and update it if needed.

**Review by pettinarip — CHANGES_REQUESTED** [review]
Looking great @vdusart! left a couple of comments. Ty

File: src/components/Layer2/Layer2Onboard.tsx [hunk 3]
```diff
@@ -163,11 +163,35 @@ const RightSelected = styled.div`
...
+  label: string
+  name: string
+  value: string
+  bridgeWallets: Array<string>
+  bridge: string
+}
+
+export interface IProps {
+  layer2DataCombined: Array<Layer2>
+  ethIcon: any
```
[the remark this answers is not in the document]
Comment (vdusart) [reply src/components/Layer2/Layer2Onboard.tsx:185]: Yes sorry for that I forgot this one...

**Comment by vdusart** [comment]
Hey @pettinarip sorry for the delay. I just fixed all your comments

**Review by pettinarip — APPROVED** [review]
Great job @vdusart thanks for the changes made here 👏🏼

**Pull request merged**